yourgov/components/Staff/AccessRequestsTable/AccessRequestsTable.tsx

```diff
@@ -0,0 +1,343 @@
+		setRejectModalOpen(false);
+	};
+
+	const isSelectAllCheckboxChecked =
+		rowSelections.length === accessRequests.length;
+	const isSelectAllCheckboxIndeterminate =
+		rowSelections.length > 0 && !isSelectAllCheckboxChecked;
+	const onChangeSelectAllRows = () => {
+		if (isSelectAllCheckboxChecked) {
+			clearRowSelections();
+		} else {
+			setRowSelections(accessRequests.map(({ id }) => id));
+		}
+	};
+
+	return (
+		<>
+			{successMessageContent && successMessageType && (
+				<SectionAlert
+					focusOnUpdate={[itemWithActiveAction || items]}
+					onClose={() => {
+						setSuccessMessageContent(undefined);
+					}}
+					title={`${
+						Array.isArray(successMessageContent)
+							? successMessageContent.length
+							: `${successMessageContent.firstName} ${successMessageContent.lastName}`
+					} access ${plural(
+						Array.isArray(successMessageContent)
+							? successMessageContent.length
+							: 1,
+						'request has',
+						'requests have'
+					)} been ${successMessageType}`}
+					tone="success"
+				>
+					<Text as="p">
+						Staff members will receive an email letting them know that their
+						request has been {successMessageType}.
+					</Text>
+				</SectionAlert>
+			)}
+			<Stack gap={0.5}>
+				{accessRequests.length === 0 ? (
+					<Stack gap={2} alignItems="flex-start" paddingY={1} role="status">
+						<Stack gap={1}>
+							<AvatarIcon size="lg" color="muted" />
+
+							<Heading type="h3" fontSize="lg">
+								There are no access requests
+							</Heading>
+
+							<Text>All access requests have been actioned.</Text>
+						</Stack>
+					</Stack>
+				) : (
+					<>
+						<Box css={{ marginLeft: mapSpacing(0.75) }}>
+							<Checkbox
+								size="sm"
+								checked={isSelectAllCheckboxChecked}
+								indeterminate={isSelectAllCheckboxIndeterminate}
+								onChange={onChangeSelectAllRows}
+							>
+								Select all rows
+							</Checkbox>
+						</Box>
+
+						<Divider />
+
+						<TableWrapper>
+							<Table aria-labelledby="access-requests-heading">
+								<TableHead>
+									<TableRow>
+										<TableHeader scope="col" width="5rem">
+											Select
+										</TableHeader>
+
+										<TableHeader scope="col">Name</TableHeader>
+
+										<TableHeader scope="col">Role</TableHeader>
+
+										<TableHeader scope="col">Status</TableHeader>
+
+										<TableHeader scope="col" width="5rem">
+											Actions
+										</TableHeader>
+									</TableRow>
+								</TableHead>
+
+								<TableBody>
+									{accessRequests.map((staffMember) => {
+										const name = `${staffMember.firstName} ${staffMember.lastName}`;
+										return (
+											<TableRow
+												key={staffMember.id}
+												selected={rowSelections.includes(staffMember.id)}
+											>
+												<TableCell>
+													<Checkbox
+														size="sm"
+														checked={rowSelections.includes(staffMember.id)}
+														onChange={() =>
```

Comment:
Changing any of the checkboxes focuses the alert 
<img width="991" alt="image" src="https://github.com/user-attachments/assets/3ab33fc6-6910-4aa0-80c3-d1696a4ee419">